x-pack/plugins/apm/public/components/app/service_node_overview/index.tsx

```diff
@@ -112,11 +119,20 @@ function ServiceNodeOverview() {
         );
       },
     },
+    {
+      name: i18n.translate('xpack.apm.jvmsTable.hostName', {
+        defaultMessage: 'Host name',
+      }),
+      field: 'hostName',
+      sortable: true,
+      render: (_, { hostName }) => getServiceNodeHostName(hostName),
```

Comment:
I'm not sure `getServiceNodeHostName` is needed. What happens if you just output it?

```suggestion
      render: (_, { hostName }) => hostName,
```

Is the problem that it prints `undefined`? In that case:

```suggestion
      render: (_, { hostName }) => hostName ?? '',
```

Reply:
👍